SMLHelper/Utility/EnumCacheManager.cs

```diff
@@ -32,6 +34,42 @@ internal EnumCacheManager(string enumTypeName, int startingIndex, IEnumerable<in
             this.enumTypeName = enumTypeName;
             this.startingIndex = startingIndex;
             this.bannedIndices = bannedIndices.ToList();
+
+            // Make sure to exclude already registered TechTypes
+            if (enumTypeName.CompareTo("TechType") == 0)
```

Comment:
This part of the code shouldn't be done in the EnumTypeCache, probably in the TechTypePatcher static constructor.